microsoft/botframework-solutions
# [TypeScript] Fix issue with installed dependencies

## Description
Fix #1269 

* Update version of `microsoft-bot-protocol-*` libraries.
* Update version of `botbuilder-*` libraries from aitemplates.
* Update `generator-botbuilder-assistant` dependencies.

## Testing Steps
1. Run `build.ts.ps1 -cliversion "1.0.1" -pkgversion "4.4.3"`
1. Check the packages generated in `./outputpackages`
